lib/config/plugins.js

```diff
@@ -66,17 +66,21 @@ module.exports = {
      * @returns {void}
      */
     define(pluginName, plugin) {
-        const pluginNameWithoutNamespace = removeNamespace(pluginName),
-            pluginNameWithoutPrefix = removePrefix(pluginNameWithoutNamespace);
-
-        plugins[pluginNameWithoutPrefix] = plugin;
+        const pluginNamespace = getNamespace(pluginName),
+            pluginNameWithoutNamespace = removeNamespace(pluginName),
+            pluginNameWithoutPrefix = removePrefix(pluginNameWithoutNamespace),
+            shortName = pluginNamespace + pluginNameWithoutPrefix;
 
         // load up environments and rules
-        Environments.importPlugin(plugin, pluginNameWithoutPrefix);
+        plugins[shortName] = plugin;
+        Environments.importPlugin(plugin, shortName);
+        Rules.importPlugin(plugin, shortName);
 
-        if (plugin.rules) {
-            rules.import(plugin.rules, pluginNameWithoutPrefix);
```

Comment:
```js
Environments.importPlugin(plugin, pluginNameWithoutPrefix);
if (plugin.rules) {
    rules.import(plugin.rules, pluginNameWithoutPrefix);
}
```

Because I felt those 2 methods are inconsistency, I modified the `rules.import` to `Rules.importPlugin`.
New form is:

```js
Environments.importPlugin(plugin, shortName);
Rules.importPlugin(plugin, shortName);
```

The modified method is a private thing; I thought this modification is no problem.